src/components/pages/LogIn.tsx

```diff
@@ -446,7 +446,7 @@ class LogInView extends React.Component<LogInProps, LogInState> {
       const redirectParams = this.props.navigation.getParam("redirectParams");
       this.props.navigation.navigate(
         redirectTo,
-        ...(redirectParams ? redirectParams : {})
+        redirectParams ? redirectParams : {}
```

Comment:
meh bummer the types don't catch this, guess there's some leakage somewhere along the line?

Reply:
I think the leakage is in React-Navigation `getParam` returning `any` type. XD